src/vs/platform/audioCues/browser/audioCueService.ts

```diff
@@ -236,10 +236,7 @@ export class AudioCueService extends Disposable implements IAudioCueService {
 		return Event.fromObservableLight(this.isCueEnabledCache.get({ cue }));
 	}
 }
-export interface IAudioCueEvent {
-	readonly cue: AudioCue;
-	readonly userGesture?: boolean;
-}
+export interface IAudioCueEvent { readonly cue: AudioCue; readonly userGesture?: boolean }
```

Comment:
I'm sorry, but with inlining I meant to replace all occurrences of `IAudioCueEvent` with the its definition (`{ readonly cue: AudioCue; readonly userGesture?: boolean }`).
We can do this now, because the is only one occurrence of it. Also, because after inlining this type the type is very local and only passed into the cache, we can omit `readonly` to make it even shorter.